implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogMeterRegistry.java

```diff
@@ -188,6 +290,26 @@ private Stream<String> writeTimer(Timer timer, Map<String, DatadogMetricMetadata
         return metrics.build();
     }
 
+    private Integer writeTimerViaStatsd(Timer timer) {
+        Meter.Id id = timer.getId();
+
+        // we can't know anything about max and percentiles originating from a function
+        // timer
+        writeMetricViaStatsd(id, "count", (double) timer.count(), Statistic.COUNT, "occurrence");
+        writeMetricViaStatsd(id, "avg", timer.mean(getBaseTimeUnit()), Statistic.VALUE, null);
+        writeMetricViaStatsd(id, "sum", timer.totalTime(getBaseTimeUnit()), Statistic.TOTAL_TIME, null);
```

Comment:
This ends up sending 3 distribution type metrics to Datadog. I'm not sure that's what we want, is it? We are aggregating Micrometer Timer recordings in Micrometer and then publishing the aggregate values once per step interval (by default, once per minute) via DogStatsD. A distribution of the 1 minute count, 1 minute sum, 1 minute average are probably not what users want. I don't think distribution type metrics will work as expected in Datadog unless all recorded values are processed by Datadog.